src/OmniSharp.DotNet/Models/DotNetProjectInformation.cs

```diff
@@ -9,12 +9,16 @@ public DotNetProjectInformation(ProjectContext projectContext, string configurat
         {
             this.Path = projectContext.RootProject.Path;
             this.Name = projectContext.ProjectFile.Name;
+            this.TargetFramework = new DotNetFramework(projectContext.TargetFramework);
 
             var outputPaths = projectContext.GetOutputPaths(configuration);
             this.CompilationOutputPath = outputPaths.CompilationOutputPath;
             this.CompilationOutputAssemblyFile = outputPaths.CompilationFiles.Assembly;
             this.CompilationOutputPdbFile = outputPaths.CompilationFiles.PdbPath;
 
+            var compilationOptions = projectContext.ProjectFile.GetCompilerOptions(targetFramework: projectContext.TargetFramework, configurationName: configuration);
```

Comment:
Sure, but we're only using it to get at a single property that this doesn't affect. Wouldn't it be very broken if I have to call an extension method to get the "right" values?